### PR TITLE
Elixir

### PR DESCRIPTION
**Pull request by pdlla**
ignore the fact that travis is failing. Will fix later D:

### CODE REVIEW

File: src/web3/web3.erl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+% TODO actually hook up to web3 XD
+
+%% how many coop chain blocks to wait before consider verified
+% -define(K, 10).
+
+%%
+init(_) ->
+  %?debugMsg("starting web3 server..."),
+  Pid = self(),
+  spawn_link(fun() -> height_poll("", Pid) end),
```
Comment (pdlla) [inline src/web3/web3.erl]: I made the mistake of doing:
spawn_link(fun() -> height_poll("", self()) end),

File: rebar.lock [hunk 2]
```diff
@@ -1 +1 @@
-[].
+[{<<"faker">>,{elixir,<<"faker">>,<<"0.6.0">>},0}].
```
Comment (antonbobkov) [inline rebar.lock]: O:

File: src/p2p/node.erl [hunk 3]
```diff
@@ -1,6 +1,6 @@
 -module(node).
 
--export([test/1]).
+-export([test/1, handle/1]).
```
Comment (antonbobkov) [inline src/p2p/node.erl]: ＼(＾O＾)／

File: src/web3/plantation.erl [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+% TODO actually hook up to web3 XD
+
+
+%%
+%% init(providerurl) ->{ok, todoprovider}.
+
+%% TODO make this non blocking
+%% TODO actually return farmers
+%%handle_call({coop_getFarmers, gameid, blocknum}, _From, _State) ->{reply, [], _State}.
```
Comment (antonbobkov) [inline src/web3/plantation.erl]: much module
such code

File: src/web3/web3.erl (hunk 1 above)
Comment (antonbobkov) [reply src/web3/web3.erl]: Lol, yeah, I've made that one too before :P

**Pull request merged**